docs/spec.json

```diff
@@ -44,6 +44,11 @@
       "description": "Open app without data",
       "default": "false"
     },
+    "quiet": {
+      "alias": "q",
+      "description": "Terse logging, limited to printing IDs where possible",
```

Comment:
To fancy? 🧐 Minimal instead?

Reply:
Be it so! I shall purge that sesquipedalian word!